pkg/tenant/apikeys.go

```diff
@@ -57,25 +57,12 @@ func (s *Server) ProjectAPIKeyList(c *gin.Context) {
 		return
 	}
 
-	// Retrieve the project from the database
-	// TODO: Check the organization namespace to determine ownership rather than retrieving the project
-	var project *db.Project
-	if project, err = db.RetrieveProject(ctx, projectID); err != nil {
-		sentry.Warn(c).Err(err).Msg("could not retrieve project from database")
-		c.JSON(http.StatusNotFound, api.ErrorResponse("project not found"))
-		return
-	}
-
-	// User should not be able to list API keys in another organization
-	if orgID.Compare(project.OrgID) != 0 {
-		sentry.Warn(c).Str("user_org", orgID.String()).Str("project_org", project.OrgID.String()).Msg("user cannot list API keys in this project")
-		c.JSON(http.StatusNotFound, api.ErrorResponse("project not found"))
-		return
-	}
+	// Verify user is on the correct organization.
+	db.VerifyOrg(c, orgID, projectID)
```

Comment:
Wow; this dramatically simplifies the code here; that's great! 

We will need to do something with what `db.VerifyOrg` returns here; e.g. if not verified we'll want to return not found as we were doing before; and that will be true for all calls to `db.VerifyOrg` -- so a little more work is necessary; but it depends on how @pdeziel responds before we can proceed.